src/main/proto/netflix/managedbatch/job.proto

```diff
@@ -396,6 +396,36 @@ message Container {
   NetworkMode networkMode = 11;
 }
 
+// BasicContainer stores the minimal data required to declare extra containers
+// to a job. This is in contrast to the Container message, which has other data
+// that are not strictly tied to the main container. For example,
+// *resources* (ram/cpu/etc) for the whole *task* are declared in the main
+// Container message, not in a basic container.
+message BasicContainer {
```

Comment:
`BasicContainer` / `extraContainers` do not convey in the name the meaning / purpose of these containers.
This message contains part of a container definition, which specifies what to execute.